src/TMap.cpp

```diff
@@ -1028,8 +1028,17 @@ bool TMap::findPath(int from, int to)
     return false;
 }
 
-bool TMap::serialize(QDataStream& ofs)
+bool TMap::serialize(QDataStream& ofs, int saveVersion)
 {
+    // clamp version values
+    if (saveVersion < 0) saveVersion = 0;
```

Comment:
Even one-liners need to have braces, because accidents happen and they can bite you.

Reply:
Sadly, although the include config file for `clang-format` will fix-up most things how we like it - one-liners like this is one thing it does not touch (although putting the braces in will correctly wrap the conditional block of code to the next line as we like)...